recipes/activiti.rb

```diff
@@ -0,0 +1,24 @@
+# create database
```

Comment:
This logic should probably be moved into `recipes/mysql-local-server.rb`, as it's something that we only need when running with a local DB (testing/demo purposes).

In a staging/production rollout, we expect the DB to be already in place (and initialised); as such, it's IMHO a responsibility of a deployment portal to init a db, not chef-alfresco